doc/users/whats_new.rst

```diff
@@ -246,6 +246,13 @@ volumetric model.
 Improvements
 ++++++++++++
 
+Add `capstyle` and `joinstyle` attributes to `Collection`
```

Comment:
capstyle and joinstyle should use *two* backquotes on both sides (this is somewhat poorly documented at http://matplotlib.org/devdocs/devel/documenting_mpl.html#formatting but the idea is that single backquotes mean "a python object that has its own entry in the docs", which is the case for Collection but not for capstyle and joinstyle, which are just parameters).

Same below.